test/spec/provider/camunda/element-templates/ReplaceBehaviorSpec.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var bootstrapModeler = require('../../../../helper').bootstrapModeler,
+    camundaModdlePackage = require('camunda-bpmn-moddle/resources/camunda'),
+    coreModule = require('bpmn-js/lib/core').default,
+    inject = require('../../../../helper').inject,
+    modelingModule = require('bpmn-js/lib/features/modeling').default,
+    propertiesPanelModule = require('lib'),
+    propertiesProviderModule = require('lib/provider/camunda'),
+    replaceModule = require('bpmn-js/lib/features/replace').default;
+
+describe('element-templates - replace behavior', function() {
+  var testModules = [
+    coreModule,
+    modelingModule,
+    replaceModule,
+    propertiesPanelModule,
+    propertiesProviderModule
+  ];
+
+  var diagramXML = require('./fixtures/replace-behavior.bpmn');
+
+  beforeEach(bootstrapModeler(diagramXML, {
+    elementTemplates: require('./fixtures/replace-behavior'),
+    modules : testModules,
+    moddleExtensions: { camunda: camundaModdlePackage }
+  }));
+
+  it('should keep the template linked', inject(function(elementRegistry, bpmnReplace) {
+    // given
+    var task = elementRegistry.get('foo');
+    var newElementData = {
+      type: 'bpmn:CallActivity'
+    };
+
+    // when
+    var newElement = bpmnReplace.replaceElement(task, newElementData);
+
+    // then
+    var businessObject = newElement.businessObject;
+    expect(businessObject.modelerTemplate).to.exist;
+  }));
+
+  it('should unlink the template', inject(function(elementRegistry, bpmnReplace) {
```

Comment:
Always make sure we have two new lines between each test

```js
it('should keep the template linked');


it('should unlink the template');
```

Reply:
Thanks, fixed!